.gitignore

```diff
@@ -6,10 +6,42 @@
 build
 dist
 dist-*
+examples
 node_modules
 npm-debug.log
 storybook-static
 yarn-error.log
 
-docs/.next
-docs/out
\ No newline at end of file
+.babelrc
+.eslintrc.cjs
+.github
+.gitignore
+.prettierrc
+.storybook
+.stylelintrc
+.stylelintrc.json
+.svgo.yml
+*.story.tsx
+*.x-story.tsx
+babel.config.cjs
+Jenkinsfile
+rollup.config.js
+tsconfig.json
+yarn.lock
+
+_process
+_storybook
+_tests
+build-styleguide
+labs
+modules
+README.md
+scripts
+src
+storybook-static
+storybook.png
+svg_src_files
+vimeo.png
+
+docs
+storybook
```

Comment:
Did you intend to commit this?

Reply:
I have **no idea** how these got added in.